documentation/assemblies/deploying/assembly-deploy-verify.adoc

```diff
@@ -3,11 +3,18 @@
 // deploying/master.adoc
 
 [id="deploy-verify_{context}"]
-= Verifying the Strimzi deployment
+= Setting up client access to the Kafka cluster
 
-Having xref:deploy-tasks_{context}[deployed Strimzi], the procedure in this section shows how to deploy example producer and consumer clients.
+Having xref:deploy-tasks_{context}[deployed Strimzi], the procedures in this section show how to:
 
-The procedure assumes a Strimzi is available and running in a Kubernetes cluster.
+* Deploy example producer and consumer clients, which you can use to verify your deployment
+* How to set up external client access to the Kafka cluster
++
```

Comment:
Remove the `+`.

Reply:
In fact, The sentence on line 13 confused me a little. Is it related to the second bullet point?

Reply:
It is related. So I'm going to leave it with the bullet.